apps/heft/.vscode/launch.json

```diff
@@ -4,6 +4,13 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+    {
+      "name": "Attach by Process ID",
+      "processId": "${command:PickProcess}",
```

Comment:
Is this useful?

Reply:
I've been using it to debug running heft with `node --inspect-brk node_modules/@rushstack/heft/bin/heft --debug build`.